programs/ziti-prox-c/CMakeLists.txt

```diff
@@ -10,7 +9,8 @@ endif()
 target_link_libraries(ziti-prox-c PUBLIC ziti subcommand)
 target_include_directories(ziti-prox-c PRIVATE ${ziti-sdk_SOURCE_DIR}/inc_internal)
 
-set(ZITI_ARCHIVE_NAME "ziti-prox-c-${ZITI_VERSION}-${CPACK_SYSTEM_NAME}.${archive_sfx}")
+set(ZITI_ARCHIVE_NAME "ziti-prox-c-${PROJECT_VERSION}-${CPACK_SYSTEM_NAME}.${archive_sfx}")
+message(${ZITI_ARCHIVE_NAME})
```

Comment:
leftover message? or maybe make the message look nicer?

Reply:
leftover debug

Reply:
removed